README.md

```diff
@@ -1,30 +1,87 @@
 # biotope-quality-gate
 
-## WARNING: This is a WIP repo for coding quality and standards on biotope
+Coding quality and standards enforcer on biotope projects.
 
-## Installing on your project
+## Installing
 - `npm i -D @biotope/quality-gate`
 
 ## Quick run
 - `biotope-quality-gate lint`
+
 which is the same as:
-- `biotope-quality-gate lint --typescript --javascript --config-file ./node_modules/@biotope/quality-gate --pattern ./`
+- `biotope-quality-gate lint --typescript --javascript --config ./node_modules/@biotope/quality-gate --pattern ./`
 
 ## Commands
 
 ### Lint
 
-Options are:
-  - -c, --config <file>       Specify a configuration file (ts or js)
-  - -p, --pattern <patterns>  Specify patterns to run against (comma separated)
-  - -t, --typescript          Include typescript files (*.ts)
-  - -j, --javascript          Include javascript files (*.js)
-  - -x, --include-jsx         Include jsx files (*.?sx)
-  - -s, --sass                Include sass files (*.scss)
-  - -f, --fix                 Try to fix any erros found
-  - -sp, --spec               Specify that the files being linted are tests (*.spec.*)
-  - -h, --help                output usage information
-
-## WARNING
-If any dependencies appear to be missing when running the software, you should use `npm install --flatten`.
-In this case, you might have installed a package that is also installing some linting tools with it.
+#### Configuration file
+This options allows you to add your own configuration file, either built from scratch or based on
+the recommended configuration given by the package (at `/` or `/config/recommended/`).
+This file, if written in typescript, will be compiled to javascript before executing.
+To use this option you can add `--config <file>` or through the shorthanded notation `-c`.
+To extend a configuration or create your own from scratch, you just need to create a typescript or
+javascript file and abide by the eslint configuration standards (for more info,
+[`read this`][link-eslint-config]).
+
+Example:
+- `biotope-quality-gate lint --config ./my-config-file.ts`
+
+#### Pattern
+This option allows you to specify where to run the linter on. Several patterns can be specified by
+separating them with commas (`,`).
+To use this option you can add `--pattern <patterns>` or through the shorthanded notation `-p`.
+
+Example:
+- `biotope-quality-gate lint --pattern src,typings`
+
+#### Typescript, Javascript, Sass and JSX
```

Comment:
Disagree. Since all the code already allows the execution of this, it should be documented.
However, you also have point. Should we just add a caveat/warning to the readme, saying it is not implemented yet?

Reply:
Offline discussion: a `warning` was added to make the unavailability of the `sass` option transparent.